tests/CMakeLists.txt

```diff
@@ -96,6 +96,10 @@ target_link_libraries(ship_streamer abieos Boost::program_options Boost::system
 
 add_test(NAME ship_test COMMAND tests/ship_test.py -v --num-clients 1 --num-requests 5000 --clean-run --dump-error-detail WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 set_property(TEST ship_test PROPERTY LABELS nonparallelizable_tests)
+if(NOT WIN32)
```

Comment:
This check can be removed: modern boost & windows supports unix sockets fine and all the original `BOOST_ASIO_HAS_LOCAL_SOCKETS` checks have already been removed

Reply:
Done, thank you!